README.md

```diff
@@ -215,6 +215,17 @@ res/
 </resources>
 ```
 
+### Watcher delay
+
+You can change the delay time until a reference is effectively considered a memory leak by providing `R.integer.__leak_canary_watch_delay_seconds` in your app:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <integer name="__leak_canary_watch_delay_seconds">10</integer>
```

Comment:
Should be `leak_canary_watch_delay_seconds` not `__leak_canary_watch_delay_seconds`. The `__` was used in the 1.3.x release.

Also, add a note that this is only available in the 1.4-SNAPSHOT version.